optuna/importance/__init__.py

```diff
@@ -3,8 +3,10 @@
 from typing import Optional
 
 from optuna.importance._base import BaseImportanceEvaluator
-from optuna.importance._fanova import FanovaImportanceEvaluator  # NOQA
-from optuna.importance._mean_decrease_impurity import MeanDecreaseImpurityImportanceEvaluator
+from optuna.importance._fanova import FanovaImportanceEvaluator
+
+# fmt: off
```

Comment:
I think we can remove these lines. Do you have specific reasons to place them?

Reply:
It's for preventing black from creating line breaks. If the following import is broken up over several lines, then flake8 will instead complain on unused variables, since `# NOQA` isn't correctly parsed. I've not encountered this problem before, do you happen to know a better workaround?

Reply:
I see. L9 exceeds 100 character limit.
Then, how about placing `# fmt: on` after the L9 to activate the black again?

Reply:
You're right it must be turned back on. Thanks, and fixed.

Reply:
Let me resolve this thread thanks to https://github.com/optuna/optuna/pull/1491#discussion_r451317817.